app/utils/emoji_utils.js

```diff
@@ -113,3 +113,42 @@ export function getEmojiByName(emojiName) {
 
     return null;
 }
+
+// Since there is no shared logic between the web and mobile app
+// this is copied from the webapp as custom sorting logic for emojis
+
+const defaultComparisonRule = (aName, bName) => {
+    return aName.localeCompare(bName);
+};
+
+const thumbsDownComparisonRule = (other) =>
+    (other === 'thumbsup' || other === '+1' ? 1 : 0);
+const thumbsUpComparisonRule = (other) => (other === 'thumbsdown' || other === '-1' ? -1 : 0);
+
+const customComparisonRules = {
+    thumbsdown: thumbsDownComparisonRule,
+    '-1': thumbsDownComparisonRule,
+    thumbsup: thumbsUpComparisonRule,
+    '+1': thumbsUpComparisonRule,
+};
+
+export function compareEmojis(emojiA, emojiB, searchedName) {
+    const aName = emojiA.name || emojiA.aliases[0];
+    const bName = emojiB.name || emojiB.aliases[0];
```

Comment:
this is causing the app to crash as `emojiA` or `emojiB` may have no `aliases` thus is trying to get the first value of an `undefined` object

```js
const aName = emojiA.name || emojiA.aliases ? emojiA.aliases[0] : emojiA;
const bName = emojiB.name || emojiB.aliases ? emojiA.aliases[0] : emojiB;
```

**Make sure to add a test case for this.**